docs/source/resources/transition_to_ft_v1.0.ipynb

```diff
@@ -0,0 +1,801 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a49ba91",
+   "metadata": {},
+   "source": [
+    "### Updating data for a dataframe in an EntitySet\n",
+    "\n",
+    "Previously to update the data associated with an Entity, users could call `Entity.update_data` and pass in the new dataframe. As an example, let's update the data in our `users` Entity:\n",
+    "```python\n",
+    "new_users_df = pd.DataFrame({\n",
+    "    'id': [3, 4],\n",
+    "    'name': ['Anne Teak', 'Art Decco']\n",
+    "})\n",
+    "\n",
+    "es['users'].update_data(df=new_users_df)\n",
+    "```\n",
+    "\n",
+    "To accomplish this task with Featuretools 1.0, we will use the `EntitySet.update_dataframe` method instead:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b45a81d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_users_df = pd.DataFrame({\n",
+    "    'id': [0, 1],\n",
+    "    'name': ['Anne Teak', 'Art Decco']\n",
+    "})\n",
+    "\n",
+    "es.update_dataframe(dataframe_name='users', df=new_users_df)\n",
+    "es['users']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "679af861",
+   "metadata": {},
+   "source": [
+    "## Defining features\n",
+    "\n",
+    "The syntax for defining features has changed slightly in Featuretools 1.0. Previously, identity features could be defined simply by passing in the variable that should be use to build the feature.\n",
+    "\n",
+    "```python\n",
+    "feature = ft.Feature(es['items']['item_price'])\n",
+    "```\n",
+    "\n",
+    "Starting with Featuretools 1.0, a similar syntax can be used, but because `es['items']` will now return a Woodwork dataframe instead of an `Entity`, we need to update the syntax slightly to access the Woodwork column. To update, simply add `.ww` between the dataframe name selector and the column selector as shown below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88902f6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature = ft.Feature(es['items'].ww['item_price'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0faf41e4",
+   "metadata": {},
+   "source": [
+    "## Defining primitives\n",
+    "\n",
+    "In earlier versions of Featuretools primitive input and return types were defined by specifying the appropriate `Varable` class. Starting in version 1.0, the input and return types are defined by Woodwork `ColumnSchema` objects. \n",
+    "\n",
+    "To illustrate this change, let's look closer at the `Age` transform primitive. This primitive takes datetime representing a date of birth and returns a numeric value corresponding to a person's age. In previous versions of Featuretools, the input type was defined by specifying the `DateOfBirth` variable type and the return type was specified by the `Numeric` variable type:\n",
+    "\n",
+    "```python\n",
+    "input_types = [DateOfBirth]\n",
+    "return_type = Numeric\n",
```

Comment:
Also out of the scope of this PR, but it seems a little odd that we'd have an `Age` Logical Type and an `Age` primitive and not have the return type be `Age` or `AgeNullable`

Reply:
If it's because this Age a float, then either this should be more specific to say `Double`, or our Age logical Type should be able to accommodate that.

Reply:
@tamargrey I think that was probably just an oversight when we updated the primitives. However, currently our `Age` logical type in Woodwork uses integer dtypes, which we would need to update to use here without loss of information.

I can update to include `Double` here for now since we are saying that is a good practice even though not strictly necessary.

Reply:
okay, seems like we have a Woodwork issue for this: https://github.com/alteryx/woodwork/issues/876

I can make a corresponding featuretools issue to use that here

Reply:
created #1642